servers/lotusmc/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "lotusmc",
+    "name": "LotusMC",
+    "store": "https://store.lotusmc.net/",
+    "addresses": [
+        "play.lotusmc.net"
```

Comment:
Remove the play. here

Reply:
alright, I went ahead and removed it from the addresses section.